src/include/pgagroal.h

```diff
@@ -210,6 +215,16 @@ struct user
    char password[MAX_PASSWORD_LENGTH]; /**< The password */
 } __attribute__ ((aligned (64)));
 
+/** @struct
+ * Defines the Prometheus connection metric
+ */
+struct prometheus_connection
+{
+   char database[MAX_DATABASE_LENGTH]; /**< The database */
```

Comment:
`struct prometheus_connection` will have a 1-to-1 relationship with `struct connection`, so this can be an empty struct to begin with